modules/nw-networkpolicy-multitenant-isolation.adoc

```diff
@@ -93,56 +88,44 @@ spec:
 EOF
 ----
 
-. If the `default` Ingress Controller configuration has the `spec.endpointPublishingStrategy: HostNetwork` value set, you must apply a label to the `default` {product-title} namespace to allow network traffic between the Ingress Controller and the project:
-
-.. Determine if your `default` Ingress Controller uses the `HostNetwork` endpoint publishing strategy:
-+
-[source,terminal]
-----
-$ oc get --namespace openshift-ingress-operator ingresscontrollers/default \
-  --output jsonpath='{.status.endpointPublishingStrategy.type}'
-----
-
-.. If the previous command reports the endpoint publishing strategy as `HostNetwork`, set a label on the `default` namespace:
-+
-[source,terminal]
-----
-$ oc label namespace default 'network.openshift.io/policy-group=ingress'
-----
-
-. Confirm that the `NetworkPolicy` object exists in your current project
-by running the following command:
-+
-[source,terminal]
-----
-$ oc get networkpolicy <policy-name> -o yaml
-----
-+
-In the following example, the `allow-from-openshift-ingress` `NetworkPolicy`
-object is displayed:
+. Optional: To confirm that the network policies exist in your current project, enter the following command:
 +
 [source,terminal]
 ----
-$ oc get -n project1 networkpolicy allow-from-openshift-ingress -o yaml
+$ oc describe networkpolicy
 ----
 +
 .Example output
-[source,terminal]
-----
-apiVersion: networking.k8s.io/v1
-kind: NetworkPolicy
-metadata:
-  name: allow-from-openshift-ingress
-  namespace: project1
-spec:
-  ingress:
-  - from:
-    - namespaceSelector:
-        matchLabels:
-          network.openshift.io/policy-group: ingress
-  podSelector: {}
-  policyTypes:
-  - Ingress
+[source,text]
```

Comment:
Are you sure you want text instead of terminal? The guidelines say to use terminal.